taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    country = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=50, unique=True)
+    password = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.username
```

Comment:
Code style issue: You should define the `Meta` class for the `Driver` model. This class is used to provide metadata for the model. It should contain `verbose_name` and `verbose_name_plural` attributes in lowercase. For example:

class Meta:
    verbose_name = "driver"
    verbose_name_plural = "drivers"